DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -1,10 +1,30 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
 
 public class ComputerGeekGame
 {
+    private string _scoreBoardfile = "DeveloperGameLeaderBoard.txt";
+    private List<KeyValuePair<int, string>> _leaderBoard = new List<KeyValuePair<int, string>>();
+
     public ComputerGeekGame()
     {
-        // To do.
+        try
+        {
+            if (File.Exists(_scoreBoardfile))
+            {
+                ReadScoresFromFile();
+            }
+            else
+            {
+                FileStream stream = File.Create(_scoreBoardfile);
+                stream.Close();
+            }
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine($"\nSomething went wrong. The following error was generated: {e.Message} Please exit the game and try again.");
```

Comment:
Also, it might be helpful to specify when it went wrong... ex: "Something went wrong when trying to load the leaderboards."

Reply:
Fixed.